### ISSUE TITLE
[CLOSED] Fix compose bug

### ISSUE THREAD

**Issue by ulfgebhardt** [Zahlungsmittel/Zahlungsmittel#197]
<a href="https://github.com/einhornimmond"><img src="https://avatars.githubusercontent.com/u/67117805?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [einhornimmond](https://github.com/einhornimmond)**
_Tuesday Apr 06, 2021 at 15:56 GMT_
_Originally opened as https://github.com/gradido/gradido/pull/138_

----

## 🍰 Pullrequest
in frontend 
depends_on:
  - nginx
  
leads to circular dependencie because nginx depens_on frontent. 
Removed it.

By the way add debug message to Login-Server after he starts visible also in docker. 

Make VUE_PATH default to /vue and copy back vue changes in nginx.conf so that on default composer-build
the new client is reachable with http://localhost/vue

----

_**[einhornimmond](https://github.com/einhornimmond)** included the following code: https://github.com/gradido/gradido/pull/138/commits_

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/ulfgebhardt"><img src="https://avatars.githubusercontent.com/u/1238238?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ulfgebhardt](https://github.com/ulfgebhardt)**
_Wednesday Apr 07, 2021 at 05:49 GMT_

----

I do not understand why nginx depends on frontend. Same applies for the login server. Nginx is the webserver and should run when we register the services. If its working I will not object, but I believe the configuration is off regarding logic and standards.

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/ulfgebhardt"><img src="https://avatars.githubusercontent.com/u/1238238?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ulfgebhardt](https://github.com/ulfgebhardt)**
_Wednesday Apr 07, 2021 at 05:52 GMT_

----

Does this solve #131 ?

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/ulfgebhardt"><img src="https://avatars.githubusercontent.com/u/1238238?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ulfgebhardt](https://github.com/ulfgebhardt)**
_Wednesday Apr 07, 2021 at 07:11 GMT_

----

We seem to need this to have the master composing again - therefore I consider to merge it and maybe make a followup issue

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/einhornimmond"><img src="https://avatars.githubusercontent.com/u/67117805?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [einhornimmond](https://github.com/einhornimmond)**
_Wednesday Apr 07, 2021 at 09:55 GMT_

----

In nginx config I use 
- proxy_pass          http://login-server:1200;
- proxy_pass          http://login-server:1201;
- proxy_pass          http://frontend:8080;
- fastcgi_pass         community-server:9000;

It seems that before starting nginx check if the services are reachable and if not, don't start. 
At least that was what I experienced as I startet with nginx. 
So it is important that this services running before nginx can start. 
If you find a another way, do it.
